common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -187,18 +187,22 @@ def test_reset_password_email_domain(self, domain_override, platform_name, send_
             req = self.request_factory.post(
                 '/password_reset/', {'email': self.user.email}
             )
+            req.is_secure = Mock(return_value=True)
             req.get_host = Mock(return_value=domain_override)
             req.user = self.user
             password_reset(req)
             _, msg, _, _ = send_email.call_args[0]
 
-            reset_msg = "you requested a password reset for your user account at {}"
+            reset_intro_msg = "you requested a password reset for your user account at {}".format(platform_name)
+            self.assertIn(reset_intro_msg, msg)
+
+            reset_link = "https://{}/"
             if domain_override:
-                reset_msg = reset_msg.format(domain_override)
+                reset_link = reset_link.format(domain_override)
```

Comment:
We lost these checks in https://github.com/edx/edx-platform/pull/15773, but I can't tell if it was on purpose, or an accident.  @ahsan-ul-haq?